core/editor/src/document/document_message_handler.rs

```diff
@@ -161,6 +162,32 @@ impl DocumentMessageHandler {
 		layers_with_indices.sort_by_key(|(_, indices)| indices.clone());
 		return layers_with_indices.into_iter().map(|(path, _)| path).collect();
 	}
+
+	/// Returns the paths to all layers in order
+	fn all_layers_sorted(&self) -> Vec<Vec<LayerId>> {
```

Comment:
This function should be combined with the selected_layers_sorted function to reduce code duplication